superset-frontend/src/SqlLab/actions/sqlLab.js

```diff
@@ -229,6 +229,9 @@ export function startQuery(query) {
 
 export function querySuccess(query, results) {
   return function (dispatch) {
+    if (!results.query) {
```

Comment:
is this line here to prevent an error on line 239? If so, I think it's still useful to store the query in line 245. I'll add a separate suggestion below if that's what your intention is.

Reply:
yeah ,to prevent query.sqlEditorId is undefined, I have tested your method, it is OK, thank you for your suggestion, I need to submit a PR again?
<img width="986" alt="image" src="https://user-images.githubusercontent.com/10183241/208219063-bbbdee11-032e-47b1-bdb2-d5ce0fa90b52.png">

Reply:
the code has been changed, please kindly review it again